katalog/fluentd/fluent-bit.conf

```diff
@@ -0,0 +1,62 @@
+[SERVICE]
+    Flush        1
+    Daemon       Off
+    Log_Level    info
+    Parsers_File parsers.conf
+    storage.path  /buffers
```

Comment:
we could add the prometheus exporter and a servicemonitor https://docs.fluentbit.io/manual/administration/monitoring

Reply:
@phisco done!